onnxscript/ir/_core.py

```diff
@@ -1533,7 +1534,7 @@ def _add_usage(self, use: Node, index: int) -> None:
 
         This is an internal method. It should only be called by the Node class.
         """
-        self._uses[(use, index)] = None
+        self._uses[use, index] = None
```

Comment:
Could you revert this change? Thanks! I will disable the check in https://github.com/microsoft/onnxscript/pull/1831 instead